modules/@apostrophecms/submitted-draft/index.js

```diff
@@ -105,24 +105,49 @@ module.exports = {
           `${self.__meta.name}:manager`,
           self.pluralLabel,
           {
-            action: 'edit',
+            action: 'publish',
             type: self.name
           },
           {
             component: 'AposSubmittedDraftAdminBarButton'
           }
         );
+        self.apos.adminBar.add(
+          `${self.__meta.name}:manager`,
+          self.pluralLabel,
+          {
+            action: 'edit',
+            type: self.name
+          },
+          {
+            icon: 'AposSubmittedDraftAdminBarIcon',
+            contextUtility: true,
+            tooltip: 'Submitted Drafts',
+            when(req) {
```

Comment:
This admin bar item is to be displayed only when the other one can't be, requiring custom logic. So I added a "when" option to admin bar items, which accepts a function that makes that determination on the fly based on `req`.